src/collection-preferences/interfaces.ts

```diff
@@ -119,6 +120,7 @@ export interface CollectionPreferencesProps<CustomPreferenceType = any> extends
    * Configures the built-in "visible content selection" preference (for example, visible columns in a table).
```

Comment:
We're still referring to the table here, could we remove that (and e.g. replace it with an example for cards)?

Reply:
I was debating that, too. Since you've brought it up as well, I'll change it.